README.md

```diff
@@ -258,6 +258,18 @@ print($record->network . "\n"); // '128.101.101.101/32'
 
 ```
 
+## Database Updates ##
+
+You can keep your databases up to date with our
+[GeoIP Update program](https://github.com/maxmind/geoipupdate/releases).
+[Learn more about GeoIP Update on our developer's
```

Comment:
Maybe "developer portal" instead as that is what the site refers to itself as? The primary reason I am commented is I wasn't sure if this should be singular or plural or possessive or not.